src/common/src/FileUpload/FileUpload.tsx

```diff
@@ -80,9 +82,15 @@ const FileUpload: React.FC<Props> = ({
 					)}
 				</FileUploadContentStyle>
 			</FileUploadStyle>
-			{files?.length > 0 && (
+			{(files?.length > 0 || (storedFiles && storedFiles.length > 0)) && (
```

Comment:
Could be written like so:
`(files?.length > 0 || storedFiles?.length > 0)`
If the condition would get any more complex I'd personally introduce a var/function for it.